setup.py

```diff
@@ -36,7 +36,7 @@
         "scikit-learn==1.1.1",
         "xgboost==1.6.1",
         "lightgbm==3.3.5",
-        "onnxruntime >=1.10.1, <1.16.0",
```

Comment:
Did you happen to test the OnnxRTDML plugin with this change?

Reply:
It would be a surprise to me if any conda-plugin had an issue with this change, since OnnxRTDML is free to use any ORT version in within_conda.py that it likes. If there's any problem we'll catch it during plugin CI/CD; I don't see a need to test it ahead of time.